### ISSUE TITLE
Why this error message: "Bad value dialog for attribute role on element dialog"?

### ISSUE THREAD

**Issue by kantoche** [validator/validator#1299]
URL being validated or code to reproduce error:

Code snippet used:
<dialog aria-labelledby="fr-modal-title-modal-1" role="dialog" id="fr-modal-1" class="fr-modal">...</dialog>

```
Bad value dialog for attribute role on element dialog
```
The other two messages are as expected warnings: 

- The dialog role is unnecessary for element dialog
- The dialog element is not supported in all browsers. Please be sure to test, and consider using a polyfill.

**Comment by scottaohara** [comment]
that is curious.  agree, the warnings are expected but not an error.

**Comment by kantoche** [comment]
I've tested with role "alertdialog" and it's safe. I don't explain myself why role "dialog" is error prone...
Does anyone know which schema or dtd is used to check theses matches?
